payments-core/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerApiRepository.kt

```diff
@@ -0,0 +1,85 @@
+package com.stripe.android.paymentsheet.repositories
+
+import com.stripe.android.Logger
+import com.stripe.android.model.ListPaymentMethodsParams
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.networking.ApiRequest
+import com.stripe.android.networking.StripeRepository
+import com.stripe.android.paymentsheet.PaymentSheet
+import kotlinx.coroutines.async
+import kotlinx.coroutines.supervisorScope
+import kotlinx.coroutines.withContext
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * A [CustomerRepository] that uses the Stripe API.
+ */
+internal class CustomerApiRepository(
```

Comment:
Looking at its usage, this class seems injectable with `publishableKey` and `stripeAccountId` Assisted, I don't have strong opinion tho, up to you:)

Reply:
I'll work on fixing the `PRODUCT_USAGE` that's coming from different places soon, will consider that too!